benchmarks/nel/scripts/datasets/dataset.py

```diff
@@ -412,10 +414,11 @@ def compare_evaluations(self, highlight_criterion: str) -> None:
 
     @classmethod
     def generate_from_id(
-        cls: Type[DatasetType], dataset_name: str, **kwargs
+        cls: Type[DatasetType], dataset_name: str, run_name: str, **kwargs
```

Comment:
It looks like you are frequently calling this method with a filler parameter. Is there a reason you don't just give the parameter a default value and omit it in calls where it's not relevant?